core/services/feeds/orm.go

```diff
@@ -204,6 +205,30 @@ func (o *orm) UpdateJobProposalStatus(ctx context.Context, id int64, status JobP
 	return nil
 }
 
+// UpdateJobProposalSpec updates the spec of a job proposal by id.
+func (o *orm) UpdateJobProposalSpec(ctx context.Context, id int64, spec string) error {
+	tx := postgres.TxFromContext(ctx, o.db)
+
+	now := time.Now()
+
+	stmt := `
```

Comment:
Tiny nitpick: you can eliminate the whitespace here and align it against the left margin